components/tyk-gateway/README.md

```diff
@@ -62,6 +62,33 @@ If you do not already have redis installed, you can use these charts provided by
 
 Follow the notes from the installation output to get connection details and password. The DNS name of your Redis as set by Bitnami is `tyk-redis-master.tyk.svc.cluster.local:6379` (Tyk needs the name including the port) You can update them in your local values.yaml file under `global.redis.addrs` and `global.redis.pass`. Alternatively, you can use `--set` flag to set it in Tyk installation. For example `--set global.redis.pass=$REDIS_PASSWORD`
 
+### Enable autoscaling
+
+This chart allows for easy configuration of autoscaling parameters. To simply enable autoscaling it's enough to add `--set gateway.autoscaling.enabled=true`. That will enable `Horizontal Pod Autoscaler` resource with default parameters (avg. CPU load at 60%, scaling between 1 and 3 instances). To customize those values you can add `--set gateway.autoscaling.averageCpuUtilization 75` or use `values.yaml` file:
+
+```yaml
+gateway:
+  autoscaling:
+    enabled: true
+    minReplicas: 3
+    maxReplicas: 30
+```
+
+Built-in rules include `gateway.autoscaling.averageCpuUtilization` for CPU utilization (set by default at 60%) and `gateway.autoscaling.averageMemoryUtilization` for memory (disabled by default). In addition to that you can define rules for custom metrics using `gateway.autoscaling.autoscalingTemplate` list:
```

Comment:
@rakkhin Any reason why you have enabled CPU utilisation by default?

Reply:
Isn't TYK CPU-bound? Makes sense to scale by CPU usage

Reply:
@jakub-bochenskiI agree with your reasoning. However since the charts don’t set resource limits by default then enabling HPA by default doesn’t make sense. I would suggest that we either set both by default or neither. 

IMO it’s better to not guess the consumer needs and leave it up to them to set these values. However I am curious to know your opinion on this

Reply:
Makes sense